components/Home/InstallTabs/LinuxPanel/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import LocalizedLink from '../../../LocalizedLink';
+import { ShellBox } from '../../../Common';
+import styles from '../index.module.scss';
+import { getNvmLatestVersion } from '../../../../util/getNvmData';
+import type { FC } from 'react';
+
+type Props = {
+  nvmVersion: string;
+};
+
+export const PureLinuxPanel: FC<Props> = ({ nvmVersion }) => {
+  const nvmInstallScriptUrl = `https://raw.githubusercontent.com/nvm-sh/nvm/${nvmVersion}/install.sh`;
+
+  return (
+    <div>
+      <ShellBox textToCopy={`curl -o- ${nvmInstallScriptUrl} | bash`}>
+        <span className={styles.installTextNoSelect}>$</span>
+        <span className={styles.installTextCommand}>curl -o- </span>
+        {nvmInstallScriptUrl}
+        <span className={styles.installTextCommand}> | bash</span>
+      </ShellBox>
+      <ShellBox textToCopy="nvm install --lts">
+        <span className={styles.installTextNoSelect}>$</span>
+        <span className={styles.installTextCommand}>nvm</span> install --lts
+      </ShellBox>
+      {/* @TODO: Verify this link when we implement content */}
```

Comment:
Do we have a process of handling todos in the code? I'm afraid this todo comment may stay here forever

Reply:
I don't know but yeah todo is dangerous

Reply:
@ovflowd Have you an idea to secure all todo ?

Reply:
What is this TODO supposed to be?